store/options.go

```diff
@@ -33,9 +33,9 @@ type Parameters struct {
 // DefaultParameters returns the default params to configure the store.
 func DefaultParameters() Parameters {
 	return Parameters{
-		StoreCacheSize: 4096,
+		StoreCacheSize: 512,
 		IndexCacheSize: 16384,
-		WriteBatchSize: 2048,
+		WriteBatchSize: 1024,
```

Comment:
lets make it 64 actually. It means we write to disk more often, but that's better then writing a lot in rare bursts

Reply:
the 2048 was actually inadequatly enormous

Reply:
done